File: java/QueryExample.java
import java.util.List;

import com.influxdb.client.InfluxDBClient;
import com.influxdb.client.InfluxDBClientFactory;
import com.influxdb.query.FluxRecord;
import com.influxdb.query.FluxTable;

public class QueryExample {

	public static void main(final String[] args) {
		String url = "https://us-west-2-1.aws.cloud2.influxdata.com";
		char[] token = "my-token".toCharArray();
		String org = "my-org";
		String bucket = "my-bucket";

		try (InfluxDBClient client = InfluxDBClientFactory.create(url, token, org, bucket)) {
			String query = "from(bucket: \"my-bucket\") |> range(start: -1h)";
			List<FluxTable> tables = client.getQueryApi().query(query, org);

			for (FluxTable table : tables) {
				for (FluxRecord record : table.getRecords()) {
					System.out.printf("%s %s: %s=%s %n",
							record.getTime(),
							record.getMeasurement(),
							record.getField(),
							record.getValue());
				}
			}
		}
	}
}

File: csharp/QueryExample.cs
using System;
using System.Threading.Tasks;
using InfluxDB.Client;

namespace Examples
{
    public class QueryExample
    {
        public static async Task Main(string[] args)
        {
            var url = "https://us-west-2-1.aws.cloud2.influxdata.com";
            var token = "my-token";
            var org = "my-org";

            using var client = InfluxDBClientFactory.Create(url, token);

            var queryApi = client.GetQueryApi();
            var query = "from(bucket:\"my-bucket\") |> range(start: -1d)";
            var tables = await queryApi.QueryAsync(query, org);
            tables.ForEach(table =>
            {
                table.Records.ForEach(record =>
                {
                    Console.WriteLine(
                        $"{record.GetTime()} {record.GetMeasurement()}: {record.GetField()}={record.GetValue()}");
                });
            });
        }
    }
}